### PR TITLE
854: Batch delete users

### PR DESCRIPTION
**Pull request by md5**
This could still be improved a bit, but I believe it's working as expected. One think in particular that could be improved would be to move the extensive model-related code in `UsersController#bulk_destroy` into the `User` model. The reason I didn't do this is that it seemed weird for the `User` model to have both the Rails-provided `destroy_all` and `delete_all` as well as a `bulk_destroy` or similarly named method.

### CODE REVIEW

**Comment by md5** [comment]
FYI, fixed the spec failure with commit 421041527ddf443b98b5757592cfe625802152b9

File: app/assets/javascripts/backbone/views/index_table_view.js.coffee [hunk 1]
```diff
@@ -18,7 +18,7 @@ class ELMO.Views.IndexTableView extends Backbone.View
     @select_all_field = this.$el.find('input[name=select_all]')
     @alert = this.$el.find('div.alert')
     @pages = this.$el.data('pages')
-    @count = this.$el.data('entries')
+    @entries = this.$el.data('entries')
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/index_table_view.js.coffee]: Zomg I love that this is a BB view now!

File: app/models/ability.rb [hunk 2]
```diff
@@ -167,7 +167,7 @@ def initialize(params)
             can :manage, UserBatch
 
             # can destroy users only if they have only one mission and it's the current mission
-            can :destroy, User do |other_user|
+            can [:bulk_destroy, :destroy], User do |other_user|
```
Comment (hooverlunch) [inline app/models/ability.rb]: Doesn't this mean that coordinators won't be able to destroy individual users anymore?

Comment (hooverlunch) [inline app/models/ability.rb]: Not sure if we need a separate ability for bulk_destroy. What is your reasoning here?

**Comment by hooverlunch** [comment]
Looks fantastic. Please merge.

**Pull request merged**